operator/Dockerfile

```diff
@@ -36,7 +36,7 @@ COPY --from=builder /workspace/armon-consul-api.tar.gz licenses/mpl_source/armon
 COPY --from=builder /workspace/hasicorp-hcl.tar.gz licenses/mpl_source/hasicorp-hcl.tar.gz
 COPY generated/admissionregistration.k8s.io_v1beta1_mutatingwebhookconfiguration_seldon-mutating-webhook-configuration.yaml /tmp/operator-resources/mutate.yaml
 COPY generated/admissionregistration.k8s.io_v1beta1_validatingwebhookconfiguration_seldon-validating-webhook-configuration.yaml /tmp/operator-resources/validate.yaml
-COPY generated/~g_v1_service_seldon-webhook-service.yaml /tmp/operator-resources/service.yaml
-COPY generated/~g_v1_configmap_seldon-config.yaml /tmp/operator-resources/configmap.yaml
+COPY generated/v1_service_seldon-webhook-service.yaml /tmp/operator-resources/service.yaml
+COPY generated/v1_configmap_seldon-config.yaml /tmp/operator-resources/configmap.yaml
```

Comment:
Which version of kustomize are you running? I assume the generated files are different for you?

Reply:
My mistake - I was in fact running a newer version (v3.8.7) but you're using v3.2.3. I just downgraded, re-ran `generate-resources` and now everything looks good.